### ISSUE TITLE
[Feature Request] Do not expand article contents

### ISSUE THREAD

**Issue by KAGEYAM4** [Reeywhaar/want-my-rss#70]
For example moz-extension://25810e7a-9f40-4d27-8e28-e84ef663e343/show.html?url=https://90sanime.com/feed/

want-my-rss expands the feed article contents and wepage becomes quite long. i want to see all articles only so that i can see type of contents and formatting provided by the rssFeed and choose to subscribe or not. With the whole article expanded, i have to scroll carefully to differentiatle between when new article start.

A preference where user can select wether they want the article inner contents to be shown or not.

**Comment by Reeywhaar** [comment]
Yep, definitely a useful feature. Will think about it. Thanks! 👍
